src/main/java/org/cactoos/func/FuncWithFallback.java

```diff
@@ -99,15 +100,15 @@
     /**
      * The fallbacks.
      */
-    private final Iterable<FallbackFrom<Y>> fallbacks;
+    private final Iterable<Fallback<Y>> fallbacks;
 
     /**
      * Ctor.
      * @param fnc The func
      * @param fbk The fallback
      */
     @SuppressWarnings("unchecked")
-    public FuncWithFallback(final Func<X, Y> fnc, final FallbackFrom<Y> fbk) {
+    public FuncWithFallback(final Func<X, Y> fnc, final Fallback<Y> fbk) {
```

Comment:
@baudoliver7 can you make this a varargs parameter as proposed here: https://github.com/yegor256/cactoos/issues/895#issuecomment-390683446